inference-engine/src/snippets/include/snippets/register_info.hpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <transformations_visibility.hpp>
+#include <ngraph/variant.hpp>
+#include <ngraph/axis_vector.hpp>
+
+namespace ngraph {
+
+template <>
+class TRANSFORMATIONS_API VariantWrapper<std::vector<size_t>> : public VariantImpl<std::vector<size_t>> {
```

Comment:
If I remember right in nGraph we already have `VariantWrapper<int64_t>` can we use it?

Reply:
I need a **vector of int64** since more than one register can be associated with an operation, while nGrraph supports only single element .

Reply:
I mean that nGraph already has [`VariantWrapper` for int64_t](https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/include/ngraph/variant.hpp#L78). In this case you can avoid introduction of new `VariantWrapper`s if you can work with `vector<int64_t>` instead of `vector<size_t>`.

Reply:
discussed offline